src/iface/interface/mod.rs

```diff
@@ -396,6 +396,9 @@ impl Interface {
     /// This function returns a boolean value indicating whether any packets were
     /// processed or emitted, and thus, whether the readiness of any socket might
     /// have changed.
+    ///
+    /// # Note
+    /// If this function returns true, there may still be packets to be transmitted or received.
```

Comment:
This note should explain *why*, because as it is the obvious solution is to do it in a loop, but this just reintroduces the original DoS.

Reply:
I expanded the why in 6492c522feb4020de3fa8aa7d509c2ce9f250af8, let me know if you have other verbiage you'd prefer and I'm happy to incorporate it :)

Reply:
Done